### PR TITLE
docs: new capitalization guidelines

### PR DESCRIPTION
**Pull request by dlimiter**
Updates capitalization guidelines. [Source](https://docs.google.com/document/d/17vst6HrIaq2UNsb4fc-uPfbNyfjJ2uxkXxiZBMS2KcA/edit)

Direct link to branch preview https://dev.cultureamp.design/dlimiter-new-capitalization-guidelines/language/grammar/#capitalization

### CODE REVIEW

File: site/docs/language/grammar.mdx [hunk 1]
```diff
@@ -72,77 +72,164 @@ As a general rule, we don’t use bold text for emphasis or to draw a user’s a
 
 ### Capitalization
 
-We use various forms of capitalization at Culture Amp. Here are the common use cases:
+Culture Amp’s general product style is sentence case. This means the first word in a sentence and proper nouns are capitalized (like names of people, places and businesses). All other words are lowercase. 
 
+We consider brand names, product names, and a few selected features to be proper nouns. Therefore, they have all words in the name capitalized. Here’s a list of agreed on proper nouns. 
 
+Agreed proper nouns (which are always capitalized):
```
Comment (lindsey-collins) [inline site/docs/language/grammar.mdx:79]: Nitpicky, but I feel like this is a bit repetitive when combined with the sentence above. We could condense these two paragraphs a bit more:

> We consider brand names, product names, and a few selected features to be proper nouns. Therefore, they have all words in the name capitalized. Here’s a list of agreed on proper nouns (which are always capitalized).

Comment (dlimiter) [reply site/docs/language/grammar.mdx:79]: Sounds good 👍

**Review by lindsey-collins — DISMISSED** [review]
Looks good, approved!

**Review by lindsey-collins — APPROVED** [review]
Approved!

**Pull request merged**